app/views/passwords/new.html.erb

```diff
@@ -1,16 +1,23 @@
-<div id="clearance" class="password-reset">
-  <h2><%= t(".title") %></h2>
+<div class="relative py-3 mx-auto max-w-lg my-4">
+  <div class="max-w-lg rounded overflow-hidden shadow-lg p-20 mb-80">
+    <div id="clearance" class="password-reset">
+      <div class="card-header py-3">
+        <h1 class="font-bold uppercase text-2xl"><%= t(".title") %></h1>
+      </div>
+      <div class="my-8">
+        <%= t(".description") %>
+      </div>
 
-  <p><%= t(".description") %></p>
-
-  <%= form_for :password, url: passwords_path do |form| %>
-    <div class="text-field">
-      <%= form.label :email %>
-      <%= form.email_field :email %>
-    </div>
-
-    <div class="submit-field">
-      <%= form.submit %>
+      <%= form_for :password, url: passwords_path do |form| %>
+        <div class="my-2">
+          <%= form.label :email, class: "font-bold" %>
+          <%= form.text_field :email, class: "w-full mt-2 p-3 rounded-lg
+          focus:outline-none focus:shadow-outline" %>
+        </div>
```

Comment:
You can use [an `email_field`][] here. This will give the user extra help and change the keyboard on some mobile devices. You can also pass an array to your `class` option. This may help in tidying up the view.

[an `email_field`]: https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-email_field

Reply:
Resolved in PR #46